testrunner-running/src/main/scala/edu/illinois/cs/statecapture/CustomSerializableConverter.java

```diff
@@ -0,0 +1,55 @@
+package edu.illinois.cs.statecapture;
+
+import java.lang.reflect.Field;
+
+import com.thoughtworks.xstream.converters.ConversionException;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.converters.reflection.SerializableConverter;
+import com.thoughtworks.xstream.converters.reflection.ReflectionProvider;
+import com.thoughtworks.xstream.core.ClassLoaderReference;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.mapper.Mapper;
+
+
+public class CustomSerializableConverter extends SerializableConverter {
```

Comment:
Similar comment as for the inheritance structure. If these concrete classes for converters override the relevant methods, is there a need for that abstract version then?